css/style.css

```
@@ -0,0 +1,81 @@
+body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  /* background-image: url("background.webp");  i don'know why i couldn't set a background with this method, i tried on empty projects, i watched tutorials and i did everything as instructed in the tutorials, is it my pc?*/
```

Because the css file is not in the same folder as the image. To reach to the folders above any folder we use `..`. If you wish to go 2 steps above you ca do `../../background.webp`. That's something I would be able to tell you on discord if you would ask the question ;) 
```suggestion
  /* background-image: URL("../background.webp");  i don'know why i couldn't set a background with this method, i tried on empty projects, i watched tutorials and i did everything as instructed in the tutorials, is it my pc?*/
```